packages/storybook/package.json

```diff
@@ -30,7 +30,7 @@
     "migrations": "./migrations.json"
   },
   "dependencies": {
-    "dotenv": "~16.3.1",
```

Comment:
I thought we still need dotenv

Reply:
create a separate pr for this, no loner needed in the package.json